docs/03_server/02_data-server/03_advanced.md

```diff
@@ -217,6 +217,97 @@ are always selected in the order they are specified in the index.
 
 The `index` property can have unique and non-unique [indexes](../../../database/data-types/index-entities/#types)
 
+Examples when: `numKeyFields > 1`
+
+Range is based on sorted records when numKeyFields is greater than 1
```

Comment:
maybe rephrase this to something like:
"The range configuration returns a set of sorted records based on the index definition and the number of fields involved (i.e. numKeyFields configuration)"